env/HERA.env

```diff
@@ -43,7 +43,7 @@ if [[ "${step}" = "prep" ]] || [[ "${step}" = "prepbufr" ]]; then
 
 elif [[ "${step}" = "preplandobs" ]]; then
 
-    export APRUN_CALCFIMS="${launcher} -n 1"
+    export APRUN_CALCFIMS="${launcher} -n 1" # TODO: Does this need threading (--cpus-per-task flag)?
```

Comment:
I don't think so, for this or the other land job

Reply:
Thanks @CoryMartin-NOAA ! I have removed those comments.